### PR TITLE
Upgrade to JDom 2, other changes

### PR DESCRIPTION
**Pull request by daveneiman**
**Resolves Issue**: (No JIRA ticket)

# What does this Pull Request do?

- Upgrade to JDom 2 from JDom 1 to resolve security issue (many source file changes)
- Update expected test output based on upgrade of MediaInfo tool
- A few other minor tweaks

# How should this be tested?
Testing must be performed from a Docker container so that generated test results are repeatable.
See explanation how to perform tests at top of [this](https://github.com/harvard-lts/fits/blob/Upgrade_JDom/docker/Dockerfile-test) file.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? yes

# Interested parties
@awoods

### CODE REVIEW

**Review by awoods — APPROVED** [review]
Visual inspection looks good and tests pass.

**Comment by awoods** [comment]
There appear to be merge conflicts.

File: src/main/java/edu/harvard/hul/ois/fits/FitsOutput.java [hunk 1]
```diff
@@ -125,46 +128,52 @@ public List<FitsMetadataElement> getTechMetadataElements() {
...
-			xpath.addNamespace("fits",Fits.XML_NAMESPACE);
-			Element node = (Element)xpath.selectSingleNode(fitsXml);
-			if(node != null) {
-				FitsMetadataElement element = buildMetdataIElements(node);
-				return element;
-			}
-		} catch (JDOMException e) {
-			//do nothing
+        XPathExpression<Element> expr = xFactory.compile("//fits:"+name, Filters.element(), null, fitsNamespace);
+//		XPathExpression<Element> expr = xFactory.compile("//fits:"+name, Filters.element(), null, fitsNamespace);
```
[the remark this answers is not in the document]
Comment (daveneiman) [reply src/main/java/edu/harvard/hul/ois/fits/FitsOutput.java]: Removed.

File: src/main/java/edu/harvard/hul/ois/fits/FitsOutput.java [hunk 2]
```diff
@@ -350,93 +358,87 @@ public void addStandardCombinedFormat() throws XMLStreamException, IOException,
...
-			for(Element element : identElements) {
-				FitsIdentity fileIdentSect = new FitsIdentity();
-
-				//get the identity attributes
-				Attribute formatAttr = element.getAttribute("format");
-				Attribute mimetypeAttr = element.getAttribute("mimetype");
-				if(formatAttr != null) {
-					fileIdentSect.setFormat(formatAttr.getValue());
+	    Namespace ns = Namespace.getNamespace("fits", Fits.XML_NAMESPACE);
+        XPathExpression<Element> expr = xFactory.compile("//fits:identity", Filters.element(), null, ns);
```
[the remark this answers is not in the document]
Comment (daveneiman) [reply src/main/java/edu/harvard/hul/ois/fits/FitsOutput.java:363]: Made tidy.

**Pull request merged**